docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -251,6 +287,49 @@ PUT _cluster/settings
   Elasticsearch compresses the response. If unset, the global
   `transport.compress` is used as the fallback setting.
 
+[float]
+[[remote-cluster-sniff-settings]]
+=== Remote cluster settings for sniff mode
+
+`cluster.remote.${cluster_alias}.seeds`::
+
+  The list of seed nodes used to sniff the remote cluster state.
+
+`cluster.remote.${cluster_alias}.node_connections`::
+
+  The number of gateway nodes to connect to for this remote cluster. The default
+  is `3`.
+
+`cluster.remote.node.attr`::
+
+  A node attribute to filter out nodes that are eligible as a gateway node in
+  the remote cluster. For instance a node can have a node attribute
+  `node.attr.gateway: true` such that only nodes with this attribute will be
+  connected to if `cluster.remote.node.attr` is set to `gateway`.
+
+[float]
+[[remote-cluster-proxy-settings]]
+=== Remote cluster settings for proxy mode
+
+`cluster.remote.${cluster_alias}.proxy_address`::
+
+  The address used for all remote connections.
+
+`cluster.remote.${cluster_alias}.proxy_socket_connections`::
+
+  The number of socket connections to open per remote cluster. The default is
+  `18`.
+ifndef::include-xpack[]
+
+`cluster.remote.${cluster_alias}.server_name`::
```

Comment:
I think this should be `ifdef`, not `ifndef`.

I'm also not sure if it's necessary to do `ifdef::include-xpack[]`. Adding `[role="xpack"]` before the setting def should suffice.

```suggestion

[role="xpack"]
`cluster.remote.${cluster_alias}.server_name`::
```